packages/devilspie/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.gnome.org/Projects/DevilsPie
+TERMUX_PKG_DESCRIPTION="Utility for matching X11 windows and performing actions on them"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Rafael Kitover <rkitover@gmail.com>"
+TERMUX_PKG_VERSION=0.23
+TERMUX_PKG_SRCURL=http://deb.debian.org/debian/pool/main/d/devilspie/devilspie_${TERMUX_PKG_VERSION}.orig.tar.xz
+TERMUX_PKG_SHA256=2dd9d4b7424ac999f20c214b0ff71e8c5836d97003d7a8a7391105cd16d36deb
+TERMUX_PKG_DEPENDS="libx11, gettext"
+TERMUX_PKG_BUILD_DEPENDS="intltool"
```

Comment:
Remove this. In Termux, `intltool` is unstable package, but repositories aren't shared. Also you can't use Termux build tools but only present in Docker image.